rabix-cli/src/main/java/org/rabix/cli/BackendCommandLine.java

```diff
@@ -247,6 +251,12 @@ protected void configure() {
                   System.exit(33);
                 }
               }
+                if (commandLine.hasOption("slurm")){
```

Comment:
In general, since SlurmModule exists, it's better to add it like TESModule is added, rather than repeat these binds. In this specific instance, if you pull the latest code from develop, you'll see that all recognized BackendModules are installed, so when you merge it, you can remove this code completely.